src/Text/Libyaml.hs

```diff
@@ -77,6 +79,14 @@ data Style = Any
            | PlainNoTag
     deriving (Show, Read, Eq, Enum, Bounded, Ord, Data, Typeable)
 
+-- @since 0.9.0
```

Comment:
This should be `-- | @since 0.9.0` to make Haddock recognize it. Bonus points would be a short sentence explaining what the type is used for.